src/bootlint.js

```diff
@@ -162,6 +187,68 @@ var LocationIndex = _location.LocationIndex;
         return runs;
     }
 
+    /**
+     * This function returns the browser window object, or null if this is not running in a browser environment.
+     * @returns {window object or null}
```

Comment:
Sorry, I didn't mean literally "or null". The proper JSDoc notation here would be something like:
`@returns {(Window|null)}`
(See http://usejsdoc.org/tags-type.html )